minifi/minifi-commons/minifi-commons-framework/src/main/java/org/apache/nifi/minifi/commons/service/FlowEnrichService.java

```diff
@@ -103,10 +107,12 @@ public byte[] enrichFlow(byte[] flowCandidate) {
             rootGroup.setInstanceIdentifier(randomUUID().toString());
         }
 
+        rootGroup.getControllerServices().forEach(cs -> cs.setScheduledState(ENABLED));
```

Comment:
This seems unrelated to the startup issue, is this specific to NiFi 2.0?

Reply:
This also caused issues because the controller services were not enabled automatically so I got validation errors. I forgot to mention this in the ticket. Because it makes no sense to disable separate components in the context of MiNiFi, I think it's safe to always enable everything.